controller3: {
  Gp_R: 10.0,
  Gp_L: 10.0,
  Gi_R: 0.9,
  Gi_L: 0.9,
  Gd_R: 0.1,
  Gd_L: 0.1,
  Gc_R: 0.0,
  Gc_L: 0.0,
  control_freq: 10,
  control_time: 0.1,
  satMin: 40.0,
  satMax: 150.0
}


controller_turn: {
  Gp_R: 0.073,
  Gp_L: 0.073,
  Gi_R: 0.0,
  Gi_L: 0.0,
  Gd_R: 0.01,
  Gd_L: 0.01,
  Gc_R: 0.00,
  Gc_L: 0.00,
  
  satMin: 0.1,
  satMax: 150.0
}
